mfthomps/CyberCIEGE-port-to-Unity3d
# Issue/317 character visits other characters

Closes #317 

This is just a first pass at getting the Staff IT guy to "visit" another user or computer.
To verify:
* Play the introduction scenario (unpaused) and hire an IT staff. He should start wandering around randomly.
* Play the Patches scenario (unpaused). Edgar, the IT guy, should get the command to visit Arlo after a few seconds and walk over to him.
* Edgar will stand there, inappropriately, forever.

Notes:
* The Staff will not interrupt their current wander target to path to a User. They have to finish their current path first.
* We are not sending the message to the server indicating the Staff have reached their destination so the scenario will never progress.
* If you delete the Staff's "visiting" field value, they will start to wander again (at least until another user-status message is received.
* This is currently only setup for the Staff IT. I haven't come across any other scenarios that command Users to visit other Users.

![image](https://user-images.githubusercontent.com/7308321/109199731-a5598e00-7754-11eb-816b-dcd7c1bbc2a3.png)
